internal/proto/buf.lock

```diff
@@ -0,0 +1,8 @@
+# Generated by buf. DO NOT EDIT.
+version: v1
+deps:
+  - remote: buf.build
+    owner: haris
+    repository: conduit-commons
```

Comment:
Note to self - make sure this points to conduitio before we merge this.

Reply:
^ Updated.